mmdet3d/datasets/s3dis_dataset.py

```diff
@@ -294,3 +285,160 @@ def _check_scene_idxs(self, scene_idx, num):
             return scene_idx
         # single idx
         return self._duplicate_to_list(scene_idx, num)
+
+
+class _S3DISDataset(Custom3DDataset):
+    """S3DIS Dataset for Detection Task."""
```

Comment:
Maybe we can add some docstring like in `_S3DISSegDataset`? For example, the paper link for users' reference. Also the docstring about each input args is required.